pages/docs/tutorials/native/interop-with-c.md

```diff
@@ -0,0 +1,140 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Interop with C Libraries"
+description: "A look at how to interop with C libraries"
+authors: Hadi Hariri 
+date: 2018-01-09
+showAuthorInfo: false
+---
+
+
+When writing native applications, oftentimes we need to access certain functionality that is not included in the Kotlin standard library, 
+such as making HTTP calls, reading and writing from disk, etc.
+
+Kotlin/Native provides us with the ability to consume standard C libraries, opening up an entire ecosystem of functionality that exists 
+for pretty much anything we could need. 
+
+In this tutorial we'll see how to create a native Kotlin application that uses the `libcurl` C library to call an HTTP endpoint. We'll cover
+how to 
+
+* [Create Kotlin Bindings](#generating-bindings)
+* [Consume a generated Kotlin API](#consuming-the-kotlin-api)
+* [Link the library into the application](#compiling-and-linking-the-library)
+
+
+## Generating Bindings
+
+An ideal scenario for interop is to call C functions as if we were calling Kotlin functions, that is, following the same signature and conventions. And that is precisely what the 
+`cinterop` tool provides us with. It takes a C library and generates the corresponding Kotlin bindings for it, which then allow us
+to use the library as if it were Kotlin code. 
+
+In order to generate these bindings, we need to create a library definition file (`def`) that contains some information about the headers we need to generate. In our case we want to use the famous `libcurl` library
+to make some HTTP calls, so we'll create a file named `libcurl.def` with the following contents
+
+```
+headers = curl/curl.h
```

Comment:
Please mention `package` argument, allowing to tell which package definition produced from def file go, like 

package = curl

Reply:
As in renaming the default package? I don't get this working. I add package = curl for instance and it still produces package libcurl.